pkg/virt-controller/services/template.go

```diff
@@ -158,6 +175,26 @@ func (t *templateService) RenderLaunchManifest(vm *v1.VirtualMachine) (*kubev1.P
 			},
 		},
 	})
+
+	transientVolumes, err := transientdisk.GenerateVolumes(vm)
```

Comment:
all this generate container/volume logic needs a test case in pkg/virt-controller/services/template.go